Makefile

```diff
@@ -1,33 +1,84 @@
 PROMBENCH_CMD        = /bin/prombench
 DOCKER_TAG = docker.io/sipian/prombench:v2.0.0
 
-deploy:
+create-nodepool:
+	printf ">> Creating NodePools"
 	$(PROMBENCH_CMD) gke nodepool create -a /etc/serviceaccount/service-account.json \
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
-		-f  components/prombench/nodepools.yaml
+		-f  /prombench/components/prombench/nodepools.yaml
```

Comment:
can't we have these relative so that the makefile can be also used without building a docker image?

Reply:
Since `WORKDIR /prombench` is in [Dockefile](https://github.com/prometheus/prombench/blob/master/Dockerfile#L19), these can be made relative